src/service/iot_config.proto

```diff
@@ -173,6 +181,36 @@ message org_create_roamer_req_v1 {
   bytes signer = 7;
 }
 
+message org_update_req_v1 {
```

Comment:
this looks odd, can you elaborate on this RPC request any why its structured with 3 sub msg types ? How does that look from the client request side of things?

Reply:
because these are all operations that update the "org" record for an iot network using organization (org record as reflected in the `org_res_v1` below)  i went back and forth on whether this should be a single update RPC or a handful of update RPCs (one for each element being updated). the `oneof` message field type convinced me to treat them as logical components of a single org update that could be executed as one-offs or by listing potential updates (the repeated list of update fields).

the rust code for the client caller would probably look something like:
```
OrgUpdateReqV1 {
    oui: 12,
    updates: vec![
        org_update_req_v1::Update::Payer(PublicKeyBinary("123DECaw532HYNJY")),
        org_update_req_v1::Update::DelegateKey(DelegateKeyUpdateV1 { delegate_key: PublicKeyBinary("1234DR564232we"), action: ActionV1::Add }),
        org_update_req_v1::Update::Devaddrs(6),
    ],
    timestamp: 1234567,
    signer: 12456abcde1234556PDERB134,
    signature: vec![], <= signed later
}
```
but on the CLI side i would _probably_ keep things simple and implement separate commands for each type of update and only construct a vec of one update operation per call. The CLI is written in rust too, so it would easily support constructing a request proto like the above but I'm guessing the actual command line input to make that happen would be more cumbersome than it's worth